test/chrome/chromeUtils.test.ts

```diff
@@ -318,6 +318,27 @@ suite('ChromeUtils', () => {
                 chromeUtils.getMatchingTargets(targets, 'http://localhost'),
                 targets);
         });
+
+        test('handles double dots', () => {
+            const targets = makeTargets('http://localhost/app', 'http://localhost/site/../folder/../app');
```

Comment:
Actually I see, this is testing that it matches `targets[0]` and doesn't match the one with dots. Sorry, I misread it.